play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/HmrcStandardPageSpec.scala

```diff
@@ -449,7 +449,7 @@ class hmrcStandardPageSpec
 
       val exitThisPageButton = document.select(".govuk-exit-this-page__button")
       exitThisPageButton                should have size 1
-      exitThisPageButton.text()       shouldBe "Exit this page"
+      exitThisPageButton.text()       shouldBe "EmergencyExit this page"
```

Comment:
might not be an issue - but should there be a space here? Seems like if the emergency is visually hidden prefix then it should have a space in between. Adding comment without checking because github is crashing my browser at the moment (got it locally checked out too but don't want to have to scroll back and find this spot if it is something to check)

Reply:
looking at a fixture it has a space, so I think we need a space adding to the exit this page after the visually hidden span component

Reply:
updated now